.circleci/config.yml

```diff
@@ -150,9 +150,9 @@ workflows:
           test-driver: kwok
           nuodb-cp-version: 2.5.0
       - functional_tests:
-          name: Functional tests (Minikube, 2.5.0)
+          name: Functional tests (Minikube, 2.6.1)
           test-driver: minikube
-          nuodb-cp-version: 2.5.0
+          nuodb-cp-version: 2.6.1
```

Comment:
This is just to avoid bugs in 2.5.0 that make this test variant more flakiness independently of this change.

Specifically, the bug that causes Domain and Databases to be in a failed state due to retriable concurrent update errors on the associated HelmApp affects this test variant.

Reply:
Yes, please. If the issue is in the webhook logic inside the operator, we should fix it.

Reply:
I remember that the transition from 2.4.x to 2.5.0 exposed it.

The following runs failed with 500 Internal Server Errors:
- https://app.circleci.com/pipelines/github/nuodb/terraform-provider-nuodbaas/617/workflows/c71b418b-cff1-46de-a272-bba8bc1a36a0/jobs/2945/tests#failed-test-0
- https://app.circleci.com/pipelines/github/nuodb/terraform-provider-nuodbaas/619/workflows/930c5f70-dfd7-461a-ad4d-5ce969c6a2fd/jobs/2956/tests#failed-test-0

Take a look at the artifacts tab, where you can find logging/stdout for the containers.

Reply:
To summarize, it looks like the issue is (1) we are not waiting to obtain the leader lease before starting the tests, which can take a very long time for some reason, and (2) the Operator crashed for some mysterious reason that we can discover if I add diagnostics to collect exited containers.